ante/gov_vote_ante.go

```diff
@@ -0,0 +1,116 @@
+package ante
+
+import (
+	errorsmod "cosmossdk.io/errors"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/authz"
+	govv1beta1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	gaiaerrors "github.com/cosmos/gaia/v15/types/errors"
+)
+
+var (
+	minStakedTokens       = sdk.NewDec(1000000) // 1000.000 uatom (or 1 atom)
```

Comment:
```suggestion
	minStakedTokens       = sdk.NewDec(1000000) // 1_000_000 uatom (or 1 atom)
```

Confusing to read. Could not tell if you added zeros after the decimal point by accident.